fullstack/src/data-source.ts

```diff
@@ -0,0 +1,17 @@
+import "reflect-metadata"
+import { DataSource } from "typeorm"
+import { ShortUrlMap } from "./entity/ShortUrlMap"
+
+export const AppDataSource = new DataSource({
+    type: "mysql",
+    host: "localhost",
+    port: 3306,
+    username: "root",
+    password: "qwe123123",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 9a6fdf9d-e73e-44d1-a8bb-f2c9eb0119e8 -->